src/locales/en.json

```diff
@@ -35,7 +35,11 @@
             "OneHourShort": "1H",
             "OneWeekShort": "1W",
             "Remind": "Remind:",
-            "ShowDetails": "show details"
+            "ShowDetails": "show details",
+            "TmcOtFlag": "Stepper driver error: OT flag set",
+            "TmcOtFlagText": "The stepper driver '{name}' has triggered the OT flag. This is an indication of a over temperature condition. This can be caused by a too high current. Please check the stepper driver settings and cooling.",
+            "TmcOtpwFlag": "Stepper driver warning: OTPW flag set",
+            "TmcOtpwFlagText": "The stepper driver '{name}' has triggered the OTPW flag. This is an indication of a over temperature condition. This can be caused by a too high current. Please check the stepper driver settings and cooling."
```

Comment:
```suggestion
            "TmcOtpwFlagText": "The stepper driver '{name}' has triggered the OTPW flag. This is an indication of an over temperature condition. This can be caused by a too high current. Please check the stepper driver settings and cooling."
```